src/internal/context/__tests__/single-tab-stop-navigation-context.test.tsx

```diff
@@ -16,10 +16,25 @@ function Button(props: React.HTMLAttributes<HTMLButtonElement>) {
 }
 
 test('does not override tab index when keyboard navigation is not active', () => {
-  renderWithSingleTabStopNavigation(<Button id="button" />);
+  renderWithSingleTabStopNavigation(<Button id="button" />, { navigationActive: false });
   expect(document.querySelector('#button')).not.toHaveAttribute('tabIndex');
 });
 
+test('does not override tab index when keyboard navigation is suppressed', () => {
```

Comment:
Keyboard navigation being "active" is the same as it being "suppressed"? I think I understand this but the terms are a bit confusing here. If I understand correctly what's happening is that there are two kinds of "navigation": the default, browser-native keyboard navigation and the single-stop grid navigation that is being implemented. Can we make this more clear even if it's just in the test descriptions?

Reply:
Thanks! When the navigation is "active" the table instructs children elements to "mute" user focus and it also listens to keyboard commands. However, some children elements can be exempted. When the focus lands on those elements the navigation is suppressed. That is to support inline-editing scenario when during the editing the table navigation should not intervene and the focus should not be muted for the elements inside of the dialog. I updated the test for it to be hopefully better clear.